modules/govuk/manifests/node/s_mirrorer.pp

```diff
@@ -1,5 +1,11 @@
 # Node defintion for mirrorer-?.management boxen
 class govuk::node::s_mirrorer inherits govuk::node::s_base {
+
   include govuk_crawler
   include nginx
+
+  include govuk_rabbitmq
```

Comment:
I think that can be achieved by doing `include govuk::apps::govuk_crawler_worker::rabbitmq` inside `govuk::apps::govuk_crawler_worker`.